### DISCUSSION TITLE
Version display on very simple setup

### DISCUSSION THREAD

**Discussion by daks** [squidfunk/mkdocs-material#3754] [Q&A] [answered]
Hello,

I just discovered mkdocs which is a nice little tool for simple documentation worflow like what I'm trying to implement:

- on CI `git clone` our repository
- `pip install` requirements
- `mkdocs build`
- push generated html files

I'm not trying to use any complex feature for now (like multiple versions), and don't want to `mkdocs serve`.

My problem is that I don't see the version displayed even with

```
# mkdocs.yml
extra:
  version: x.y.z
```

Is it possible to do?

_update: I found some issues/discussions talking about version display but most were asking in a multi-versioning setup or using mike, and none asking in my simple use case_

**Comment by squidfunk** [comment, marked as answer]
You currently cannot set a version string explicitly in `mkdocs.yml`. You need to use a tool that supports proper versioning, and currently, the only tool supported is `mike`, as is [noted in the documentation](https://squidfunk.github.io/mkdocs-material/setup/setting-up-versioning/). The reason for this is that proper version support needs a lot more logic than just setting a version, which is why we decided to natively support `mike`.

You could in theory add support yourself by using theme extension though.

**Reply by daks** [reply]
OK, so it means that this template is for advanced uses and is not suitable for simple ones like mine. I like the simple usecase of mkdocs and wanted to use a more user-friendly template but it seems that this one is not the one.
Thanks for your answer.
